### PR TITLE
Minor optimisations, adding robots.txt, corrections

### PR DESCRIPTION
**Pull request by chriscoffee**
- Noticed that we didn't have the discord invite URL on the footer of the aarch64.com site, so I've added that, along with the svg and such.
- ~Added robots.txt, defaults back to / if environment isn't production.~ Removed for now as I didn't think about deploy previews and the fix I put in didn't work.
- Added some headers, we should add others i.e Permissions-Policy and CSP!
- Netwise dot com URL has issues, used the UK version.
- Added Netwise to about section as it's missing there along with the other sponsors.
- Made fonts quicker
- Made images smaller whilst keeping quality.

### CODE REVIEW

File: components/common/Header.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const Header = (props) => (
       <Link href="/">
         <a>
           <object data="/logo.svg" type="image/svg+xml">
-            <img src="/logo.png" alt="Arm-64.com Logo" />
+            <img src="/logo.png" alt="aarch-64.com Logo" />
```
Comment (dragstor) [inline components/common/Header.tsx]: aarch64.com 😄

File: next.config.js [hunk 2]
```diff
@@ -2,6 +2,39 @@ const withPlugins = require("next-compose-plugins");
...
+            key: 'Strict-Transport-Security',
+            value: 'max-age=31536000; includeSubDomains; preload',
+          },
+          {
+            key: 'X-DNS-Prefetch-Control',
+            value: 'on',
+          },          
+          {
+            key: 'Permissions-Policy',
+            value: 'geolocation=(self "https://aarch64.com"), microphone=()',
```
Comment (dragstor) [inline next.config.js]: why do we need this permission?

File: components/common/Header.tsx (hunk 1 above)
Comment (chriscoffee) [reply components/common/Header.tsx]: Nice spot, not that it was right originally 😂 I'll correct now!

File: next.config.js (hunk 2 above)
Comment (chriscoffee) [reply next.config.js]: I guess we don't, I removed that anyway!

**Review by dragstor — APPROVED** [review]

File: components/common/Footer.tsx [hunk 3]
```diff
@@ -42,6 +43,14 @@ const Footer = () => (
           <GitHubIcon />
           <span>Github Link</span>
         </a>
+        <a
+          href="https://discord.gg/8MfNdGK"
```
Comment (sr229) [inline components/common/Footer.tsx:47]: I assume we can change this to a Vanity URL instead. We got one months after this PR was made.

**Review by sr229 — CHANGES_REQUESTED** [review]

File: components/common/Footer.tsx (hunk 3 above)
Comment (brunomiguel) [reply components/common/Footer.tsx:47]: It's https://discord.gg/foss

Comment (brunomiguel) [reply components/common/Footer.tsx:47]: Also, Netwise is no longer a sponsor

**Comment by mplscorwin** [comment]
I would prefer to promote Liberal.Chat vs Discord, who is neither FOSS nor
a sponsor.

Why not direct folk to our support portal instead?  support.fossho.st

I can live with both/all.

**Pull request closed without merging**